virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="virt-managing-vms-openshift-pipelines"]
+= Managing virtual machines with OpenShift Pipelines
+include::_attributes/common-attributes.adoc[]
+:context: virt-managing-vms-openshift-pipelines
+
+toc::[]
+
+xref:../../cicd/pipelines/understanding-openshift-pipelines.adoc#understanding-openshift-pipelines[Red Hat OpenShift Pipelines] is a Kubernetes-native CI/CD framework that allows developers to design and run each step of the CI/CD pipeline in its own container.
+
+The Tekton Tasks Operator (TTO) integrates {VirtProductName} with OpenShift Pipelines. TTO includes cluster tasks and example pipelines that allow you to:
```

Comment:
ISG says the following about "allow":

Use with caution 
allow   verb
Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."